.github/workflows/trigger-anaconda.yml

```diff
@@ -0,0 +1,63 @@
+# Anaconda's tests are "special" (require booting boot.iso, can't run in
+# tmt/Testing Farm), so we need to run them in Cockpit's CI. But we still want
+# to re-use the packit COPR, which is much easier to consume by Anaconda's
+# tests than building cockpit in Anaconda.
+# This workflow checks if the PR affects Anaconda (changes to the bridge or
+# Storage page), polls the packit COPR until it has the current PR version
+# available, and then test-triggers a "cockpit PR" scenario.
+
+name: anaconda
+on: pull_request_target
+jobs:
+  trigger:
+    runs-on: ubuntu-22.04
+    # the default workflow token cannot read our org membership, for deciding who is allowed to trigger tests
+    environment: gh-cockpituous
+    container: registry.fedoraproject.org/fedora:rawhide
+    # this polls for a COPR build, which can take long
+    timeout-minutes: 120
+
+    steps:
+      - name: Install dependencies
+        run: |
+          dnf install -y git-core dnf-plugins-core || {
+            sleep 60
+            dnf install -y git-core dnf-plugins-core
+          }
+
+      - name: Clone target branch
+        uses: actions/checkout@v3
+
+      # https://github.blog/2022-04-12-git-security-vulnerability-announced/
+      - name: Pacify git's permission check
+        run: git config --global --add safe.directory /__w/cockpit/cockpit
+
+      - name: Check if PR affects Anaconda
+        id: affected
+        run: |
+          git fetch origin refs/pull/${{ github.event.number }}/head
+          git diff --stat --exit-code HEAD..FETCH_HEAD -- src/cockpit pkg/storaged >&2 || echo "changed=true" >> "$GITHUB_OUTPUT"
```

Comment:
Sounds like it's better accomplished by way of `on.pull_request_target.paths`: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
Thanks for pointing out! This is so much nicer indeed, then the workflow doesn't even start nor show up in the checks list for PRs which don't affect anaconda. I updated the workflow accordingly, and tested both ways in #19459 again.